examples/rl/racing/Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 RUN pip install --upgrade pip wheel
 
 # Setup SUMO.
-RUN pip install eclipse-sumo==1.10.0
+RUN pip install eclipse-sumo==1.12.0
```

Comment:
Examples are tested to work with the given dependencies and are defined by their `setup.py`. In general, it is better to leave examples untouched unless they have been tested again.

Reply:
Ok, I've reverted this.